Core/include/Acts/Visualization/GeometryView3D.hpp

```diff
@@ -81,6 +82,38 @@ struct GeometryView3D {
                          const Transform3& transform = Transform3::Identity(),
                          const ViewConfig& viewConfig = s_viewVolume);
 
+  /// Helper method to draw portals objects
+  ///
+  /// @param [in,out] helper The visualization helper
+  /// @param portal The portal to be drawn
+  /// @param gctx The geometry context for which it is drawn
+  /// @param transform An option additional transform
+  /// @param connected The config for connected portals
+  /// @param unconnected The config for unconnected portals
+  static void drawPortal(IVisualization3D& helper,
+                         const Experimental::Portal& portal,
+                         const GeometryContext& gctx,
+                         const Transform3& transform = Transform3::Identity(),
+                         const ViewConfig& connected = ViewConfig({0, 255, 0}),
+                         const ViewConfig& unconnected = ViewConfig({255, 0,
+                                                                     0}));
+
+  /// Helper method to draw DetectorVolume objects
+  ///
+  /// @param [in,out] helper The visualization helper
+  /// @param volume The DetectorVolume to be drawn
+  /// @param gctx The geometry context for which it is drawn
+  /// @param transform An option additional transform
+  /// @param connected The config for connected portals
+  /// @param unconnected The config for unconnected portals
+  static void drawDetectorVolume(
+      IVisualization3D& helper,
+      const Acts::Experimental::DetectorVolume& volume,
+      const GeometryContext& gctx,
+      const Transform3& transform = Transform3::Identity(),
+      const ViewConfig& connected = ViewConfig({0, 255, 0}),
+      const ViewConfig& unconnected = ViewConfig({255, 0, 0}));
```

Comment:
Fo be consistent with the `cpp` and make clang-tidy happy:

```suggestion
  /// @param disconnected The config for unconnected portals
  static void drawDetectorVolume(
      IVisualization3D& helper,
      const Acts::Experimental::DetectorVolume& volume,
      const GeometryContext& gctx,
      const Transform3& transform = Transform3::Identity(),
      const ViewConfig& connected = ViewConfig({0, 255, 0}),
      const ViewConfig& disconnected = ViewConfig({255, 0, 0}));
```